cura/PrinterOutput/PrinterOutputModel.py

```diff
@@ -238,3 +256,14 @@ def canControlManually(self):
         if self._controller:
             return self._controller.can_control_manually
         return False
+
+    # Returns the configuration (material, variant and buildplate) of the current printer
+    @pyqtProperty(QObject, notify = configurationChanged)
+    def printerConfiguration(self):
+        return self._printer_configuration
+
+    def _updatePrinterConfiguration(self):
+        self._printer_configuration.printerType = self._type
```

Comment:
Same comment as above about variables being able to be out of sync.

Reply:
Same as above :-)